### DISCUSSION TITLE
#42412 - Add Foster Marketplace to Phantom's DApp Registry

### DISCUSSION THREAD

**Discussion by dragondotfun** [phantom/docs#114] [Help & Support] [answered]
### Summary

Hi Phantom! My name is Andrew, Co-Founder of Foster Marketplace. I created a ticket back on June 16th to get our marketplace added to your dapp registry. I was advised by Harry at Phantom Support to open a developer discussion to closely monitor the status of this request. The request ticket ID is #42412 - Please let me know if you need anything else from my end.

Thank you in advance for your assistance.

### Example

https://fostermarketplace.app/

### Steps to Reproduce

Right now, the `from` address in phantom displays the auction house wallet address ![image](https://github.com/phantom/docs/assets/93629141/36a5dd7f-a7bb-49f5-9cf3-7d63439318b3)

What we are looking to achieve is having "Foster Marketplace" be displayed instead, similar to how EA is setup 
![image](https://github.com/phantom/docs/assets/93629141/f44a54e4-1c7e-46d2-860a-fcd85c06f61a)

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hey @StealthofFoster,

Thank you for posting your request here. I am also now aware of your support ticket. I will follow up with you on this shortly.

**Comment by dragondotfun** [comment]
Thank you so much, Adam.

**Comment by dragondotfun** [comment]
Hey Adam,

Just wanted to follow up on this again as it doesn’t seem that Foster
Marketplace was added to Phantom’s dApp registry

Thank you,

Andrew Hanna, MBA, B.S.
T: (480) 845 - 2405
F: (844) 712- 9035

**Reply by adamdelphantom** [reply]
Hey Andrew,

Thanks for your patience! I believe I have a path forward to get the dApp name showing in activity. I'm going to try to get it done this week. I'll let you know once I make some progress.

**Comment by dragondotfun** [comment]
Sounds great, thanks for the update!

**Comment by dragondotfun** [comment]
Hi Adam,

Any luck?

**Reply by adamdelphantom** [reply]
Not this week, sorry. Will try to get this taken care of next week prior to the US holiday. I'll let you know as soon as I have a proper eta.

**Comment by dragondotfun** [comment]
Hi Adam,

I still haven’t heard back from you mate, any reason for the extended delay?

We opened this ticket back in June.

Thank you,

Andrew Hanna, MBA, B.S.
T: (480) 845 - 2405
F: (844) 712- 9035

**Reply by adamdelphantom** [reply]
Hey Andrew,

Apologies for the continued delay. Would you mind sharing an image file to use as the foster marketplace logo, please?

**Comment by dragondotfun** [comment]
No problem, I appreciate your support on this.

I've attached the Foster Marketplace logo.

**Reply by adamdelphantom** [reply]
I think you're using email to respond and the attachment didn't come through. You might need to post directly to GitHub. Additionally can you supply the relevant Solana program ID's for Foster Marketplace, please?

**Reply by dragondotfun** [reply]
Sorry about that. Here is the logo 
![Foster_LIFT_3D_Logo](https://github.com/phantom/docs/assets/93629141/ad5d7647-bc70-4f10-b8d8-2addf030c9de)

We are currently using Metaplex Auction House (2), can you use a specific key in the transaction instead of the program id?

Auction house (Fixed Price Listings): BFuH1xRrnqhUKL8S7mNPJSanARLprsExAPJvDXnS6uVi
Auction house (Timed Auctions): 82zoX5Ctbc4P85jyYUqrqtW2jScDL7RJrKqRHta9EbFd

**Reply by adamdelphantom** [reply]
Thanks for providing this! I will let you know as soon as I have an eta. Thanks again for your patience!

**Reply by dragondotfun** [reply]
Hey Adam, it's been a hot minute was this something you were still looking into?

**Reply by adamdelphantom** [reply, marked as answer]
@StealthofFoster My sincere apologies. You've been incredibly patient and I appreciate it. I'm getting this done ASAP would you also mind sending me a DM on twitter or telegram please? Same handle adamdelphantom.

**Reply by dragondotfun** [reply]
I appreciate that man, I just sent you a DM on twitter!
